Cannot execute method GetMyName

**Describe the bug**
When executing the GetMyName method, an exception is thrown: "Exception in thread "main" java.lang.RuntimeException: Unable to deserialize response"

**To Reproduce**
Steps to reproduce the behavior:
```
TelegramClient telegramClient = new OkHttpTelegramClient(botToken);
GetMyName getMyName = GetMyName.builder()
                    .build();
telegramClient.execute(getMyName)
```
1. Run the code
2. See error

**Expected behavior**
Return BotName

**Desktop (please complete the following information):**
 - macOS
 - Browser Chrome
 - Version 15.3 Beta(24D5034f)

**Additional context**
The full stack trace is as follows:
```
Exception in thread "main" java.lang.RuntimeException: Unable to deserialize response
	at com.example.telegram.Telegram.<init>(Telegram.java:62)
	at com.example.telegram.Telegram.getInstance(Telegram.java:71)
	at com.example.Main.main(Main.java:27)
Caused by: Unable to deserialize response
	at org.telegram.telegrambots.meta.api.methods.botapimethods.PartialBotApiMethod.deserializeResponseInternal(PartialBotApiMethod.java:66)
	at org.telegram.telegrambots.meta.api.methods.botapimethods.PartialBotApiMethod.deserializeResponse(PartialBotApiMethod.java:43)
	at org.telegram.telegrambots.meta.api.methods.name.GetMyName.deserializeResponse(GetMyName.java:43)
	at org.telegram.telegrambots.meta.api.methods.name.GetMyName.deserializeResponse(GetMyName.java:22)
Caused by: Unable to deserialize response

	at org.telegram.telegrambots.client.okhttp.OkHttpFutureCallback.onResponse(OkHttpFutureCallback.java:35)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `org.telegram.telegrambots.meta.api.objects.name.BotName` (although at least one Creator exists): cannot deserialize from Object value (no delegate- or property-based Creator)
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 22] (through reference chain: org.telegram.telegrambots.meta.api.objects.ApiResponse["result"])
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `org.telegram.telegrambots.meta.api.objects.name.BotName` (although at least one Creator exists): cannot deserialize from Object value (no delegate- or property-based Creator)

	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:63)
	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1754)
	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1379)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1508)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:348)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:310)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
	at org.telegram.telegrambots.meta.api.methods.botapimethods.PartialBotApiMethod.deserializeResponseInternal(PartialBotApiMethod.java:59)
	... 8 more
```

--- a/telegrambots-client/src/test/java/org/telegram/telegrambots/client/TestTelegramClientIntegration.java
+++ b/telegrambots-client/src/test/java/org/telegram/telegrambots/client/TestTelegramClientIntegration.java
@@ -14,6 +14,13 @@ import org.mockito.MockitoAnnotations;
 import org.telegram.telegrambots.client.okhttp.OkHttpTelegramClient;
 import org.telegram.telegrambots.meta.TelegramUrl;
 import org.telegram.telegrambots.meta.api.methods.botapimethods.PartialBotApiMethod;
+import org.telegram.telegrambots.meta.api.methods.commands.GetMyCommands;
+import org.telegram.telegrambots.meta.api.methods.description.GetMyDescription;
+import org.telegram.telegrambots.meta.api.methods.description.GetMyShortDescription;
+import org.telegram.telegrambots.meta.api.methods.description.SetMyDescription;
+import org.telegram.telegrambots.meta.api.methods.description.SetMyShortDescription;
+import org.telegram.telegrambots.meta.api.methods.name.GetMyName;
+import org.telegram.telegrambots.meta.api.methods.name.SetMyName;
 import org.telegram.telegrambots.meta.api.methods.send.SendAnimation;
 import org.telegram.telegrambots.meta.api.methods.send.SendAudio;
 import org.telegram.telegrambots.meta.api.methods.send.SendDocument;
@@ -24,16 +31,22 @@ import org.telegram.telegrambots.meta.api.methods.send.SendVideo;
 import org.telegram.telegrambots.meta.api.methods.send.SendVideoNote;
 import org.telegram.telegrambots.meta.api.methods.send.SendVoice;
 import org.telegram.telegrambots.meta.api.objects.InputFile;
+import org.telegram.telegrambots.meta.api.objects.commands.BotCommand;
+import org.telegram.telegrambots.meta.api.objects.description.BotDescription;
+import org.telegram.telegrambots.meta.api.objects.description.BotShortDescription;
 import org.telegram.telegrambots.meta.api.objects.message.Message;
+import org.telegram.telegrambots.meta.api.objects.name.BotName;
 import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
 import org.telegram.telegrambots.meta.exceptions.TelegramApiRequestException;
 
 import java.io.File;
 import java.io.Serializable;
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.*;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class TestTelegramClientIntegration {
+class TestTelegramClientIntegration {
     private MockWebServer webServer;
 
     private static final String TOKEN = "testToken";
@@ -166,6 +179,84 @@ public class TestTelegramClientIntegration {
 
         Message parsedMessage = client.execute(method);
         assertEquals(responseMessage, parsedMessage);
+    }
+
+    @Test
+    void testGetMyName() throws TelegramApiException {
+        var method = new GetMyName();
+
+        var expected = new BotName(TestData.BOT_USER.getFirstName());
+        mockMethod(method, expected);
+
+        var actual = client.execute(method);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void testSetMyName() throws TelegramApiException {
+        var method = new SetMyName();
+        method.setName(TestData.BOT_USER.getFirstName());
+
+        mockMethod(method, Boolean.TRUE);
+
+        assertTrue(client.execute(method));
+    }
+
+    @Test
+    void testGetMyDescription() throws TelegramApiException {
+        var method = new GetMyDescription();
+
+        var expected = new BotDescription("description");
+        mockMethod(method, expected);
+
+        var actual = client.execute(method);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void testSetMyDescription() throws TelegramApiException {
+        var method = new SetMyDescription();
+        method.setDescription("description");
+
+        mockMethod(method, Boolean.TRUE);
+
+        assertTrue(client.execute(method));
+    }
+
+    @Test
+    void testGetMyShortDescription() throws TelegramApiException {
+        var method = new GetMyShortDescription();
+
+        var expected = new BotShortDescription("short");
+        mockMethod(method, expected);
+
+        var actual = client.execute(method);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void testSetMyShortDescription() throws TelegramApiException {
+        var method = new SetMyShortDescription();
+        method.setShortDescription("description");
+
+        mockMethod(method, Boolean.TRUE);
+
+        assertTrue(client.execute(method));
+    }
+
+    @Test
+    void testGetMyCommands() throws TelegramApiException {
+        var method = new GetMyCommands();
+
+        var expected = new ArrayList<BotCommand>();
+        expected.add(BotCommand.builder()
+                .command("command")
+                .description("description")
+                .build());
+        mockMethod(method, expected);
+
+        var actual = client.execute(method);
+        assertEquals(expected, actual);
     }
 
     @Test

--- a/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/description/BotDescription.java
+++ b/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/description/BotDescription.java
@@ -7,6 +7,7 @@ import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
+import lombok.extern.jackson.Jacksonized;
 import org.telegram.telegrambots.meta.api.interfaces.BotApiObject;
 
 /**
@@ -20,7 +21,8 @@ import org.telegram.telegrambots.meta.api.interfaces.BotApiObject;
 @ToString
 @AllArgsConstructor
 @Builder
-public class BotDescription implements BotApiObject  {
+@Jacksonized
+public class BotDescription implements BotApiObject {
     private static final String DESCRIPTION_FIELD = "description";
 
     /**

--- a/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/description/BotShortDescription.java
+++ b/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/description/BotShortDescription.java
@@ -7,6 +7,7 @@ import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
+import lombok.extern.jackson.Jacksonized;
 import org.telegram.telegrambots.meta.api.interfaces.BotApiObject;
 
 /**
@@ -20,7 +21,8 @@ import org.telegram.telegrambots.meta.api.interfaces.BotApiObject;
 @ToString
 @AllArgsConstructor
 @Builder
-public class BotShortDescription implements BotApiObject  {
+@Jacksonized
+public class BotShortDescription implements BotApiObject {
     private static final String SHORT_DESCRIPTION_FIELD = "short_description";
 
     /**

--- a/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/name/BotName.java
+++ b/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/name/BotName.java
@@ -7,6 +7,7 @@ import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
+import lombok.extern.jackson.Jacksonized;
 import org.telegram.telegrambots.meta.api.interfaces.BotApiObject;
 
 /**
@@ -20,7 +21,8 @@ import org.telegram.telegrambots.meta.api.interfaces.BotApiObject;
 @ToString
 @AllArgsConstructor
 @Builder
-public class BotName implements BotApiObject  {
+@Jacksonized
+public class BotName implements BotApiObject {
     private static final String NAME_FIELD = "name";
 
     /**

